Xamarin.Essentials/Email/Email.android.cs

```diff
@@ -34,7 +34,12 @@ static Intent CreateIntent(EmailMessage message)
                 if (message?.BodyFormat == EmailBodyFormat.Html)
                 {
                     ISpanned html;
-                    if (Platform.HasApiLevel(BuildVersionCodes.N))
+#if __ANDROID_24__
+                    var hasN = Platform.HasApiLevel(BuildVersionCodes.N);
+#else
+                    var hasN = false;
+#endif
+                    if (hasN)
```

Comment:
For the amount of times we do this, I would maybe create specific properties in Android's Platform - `HasApiLevelN` so that we can keep this `#if` stuff in one place. I can just see us either forgetting to use the `#if` or not properly doing something else.

Might be good to do this for the other major levels - such as M and P

Reply:
I like that idea :)